src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body,
+.input-font {
+  font-family: "Avenir";
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-bar::before {
+  content: "";
+  position: absolute;
+  background-image: url(images/Search.svg);
+}
+
+.search-bar__input {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar__input:hover {
+  background: #fff;
```

Comment:
if you have some styles on the element, there is no need to write them again for hover state or some other state